@navikt/core/react/src/util/i18n/locales/en.ts

```diff
@@ -37,4 +37,7 @@ export default {
       labelSuffix: "delete",
     },
   },
+  Modal: {
+    close: "Close",
```

Comment:
```suggestion
    close: "Close dialog",
```
Would this be more descriptive? Or do most people not know the concept of a "dialog" so "close" is better?

Reply:
We had "Close modal" before, but David complained about it and said it was redundant (and annoying iirc 😅). The SR announces that you are in a dialog, so I think it's implied that "close" closes the dialog.

When it comes to seeing people, I think you are right in that many people probably doesn't understand "dialog". But I think most people are familiar with windows (not the OS but the UI concept) and the X button in the corner of many of them.